### ISSUE TITLE
Feature counts for paired end reads

### ISSUE THREAD

**Issue by Fred-White94** [BleekerLab/snakemake_rnaseq#10]
Despite there being the function to determine from the sample file whether or not you have single or paired end reads this does not carry over to the featurecounts rule which is only set up for single end (and fails to properly count the reads for paired end samples).

The use of -p option for paired end reads should be linked to the initial function. Easiest way would be to add in another if else for this rule with and without the -p. I cannot think of a way to include it with minimal code (i.e. avoiding another if else).

You could make the sample_is_single_end output a wildcard {endedness} which ="" for single ended and = "-p" for paired end?

Cheers!

**Comment by mgalland** [comment]
In the BAM file, either single-end or paired-end reads should be present right? So as long as one alignment overlaps with the GTF file and the `-t / -g` features to be counted, it should be taken into account in the raw counts being outputed. 

Or do you have a specific option in featureCounts for single/paired-end reads that I missed?

**Comment by Fred-White94** [comment]
Yes I asssumed so too. But when using feature counts to summarise per exon the `-p` argument (to signify paired end) needs to included in order for it to properly assign the reads, otherwise they seem to all map to the first/last exon or not at all. 

I will do another run with paired end summarised for genes today and get back to you whether or not it is specifically a problem for exon level quantifications.
